### ISSUE TITLE
issue with dashboard

### ISSUE THREAD

**Issue by juancarlosfarah** [graasp/graasp-desktop#343]
TypeError: Cannot read property 'actionAccessibility' of undefined

![image](https://user-images.githubusercontent.com/1707188/95833830-78c04f00-0d3c-11eb-8a63-d33872d3dc72.png)
